http/src/main/resources/reference.conf

```diff
@@ -56,6 +56,66 @@ gce {
   gceReservedMemory = 1g
 }
 
+vpc {
+  highSecurityProjectNetworkLabel = "vpc-network-name"
+  highSecurityProjectSubnetworkLabel = "vpc-subnetwork-name"
+  networkName = "leonardo-network"
+  networkTag = "leonardo"
+  autoCreateSubnetworks = false
+  # note the below 3 fields are not used if autoCreateSubnetworks is true
+  subnetworkName = "leonardo-subnetwork"
+  subnetworkRegion = "us-central1"
+  subnetworkIpRange = "10.128.0.0/20"
```

Comment:
This is the ip range when creating the subnet. It can be any ip range but I chose the range for the `us-central1` subnet in [auto mode](https://cloud.google.com/vpc/docs/vpc#ip-ranges).

By the way, I experimented with auto mode and custom mode subnet creation ([docs](https://cloud.google.com/vpc/docs/vpc#vpc_networks_and_subnets)). Either way works -- but this PR is using custom mode because:
* we only need subnets in 1 region (auto mode creates one in _every_ region)
* we have control over the IP range which is useful for creating fw rules
* [these docs](https://cloud.google.com/vpc/docs/vpc#auto-mode-considerations) recommend using custom mode for production

Maybe I'll synthesize this in a comment in the `reference.conf` file..